packages/monaco-contributions/common.js

```diff
@@ -0,0 +1,30 @@
+export const EVALUATE_KEYWORD = [
```

Comment:
统一下术语吧，以前叫「求值占位符」或「求值表达式」，现在叫「表达式」
```suggestion
export const BRICK_NEXT_EXPRESSION_KEYWORD = [
```